Source/WTF/wtf/RetainPtr.h

```diff
@@ -323,6 +323,7 @@ template<typename T> constexpr RetainPtr<T> adoptCF(T CF_RELEASES_ARGUMENT ptr)
 #ifdef __OBJC__
 template<typename T> inline RetainPtr<typename RetainPtr<T>::HelperPtrType> adoptNS(T NS_RELEASES_ARGUMENT ptr)
 {
+    static_assert((std::is_convertible<T, id>::value), "Don't use adoptNS with Core Foundation pointer types, use adoptCF.");
```

Comment:
Please use `std::is_convertible_v` and also omit unneeded extra set of parentheses.

Reply:
I will do you one better, I will change all the ::value to _v